v2/contribute/decisions/reactsdk/0002-custom-field-tabindex-issue.mdx

```diff
@@ -0,0 +1,45 @@
+---
+id: "0002"
+title: Fix input tabbing issue in main-website auth route
+hide_title: true
+---
+
+import DecisionHeader from "/src/components/decisionLogs/DecisionHeader"
+import ArgumentPro from "/src/components/decisionLogs/ArgumentPro"
+import ArgumentCon from "/src/components/decisionLogs/ArgumentCon"
+import ArgumentNeut from "/src/components/decisionLogs/ArgumentNeut"
+
+# Fix input tabbing issue in main-website auth route
+
+<DecisionHeader status="proposed" lastUpdate="2023-01-17" created="2023-01-17" deciders={["rishabhpoddar", "porcellus"]} proposedBy={["alisher-aituarov"]} />
+
+## Context and Problem Statement
+
+When custom field is added to sign up form and the order of these fields are changed using CSS keyboard navigation between inputs is broken. See related [issue](https://github.com/supertokens/supertokens-auth-react/issues/634)
+
+## Considered Options
+
+* Change tabindex value in main-website repo itself
+* Make Recipe.init custom field(formFields) accept additional order property
+
+
+## Decision Outcome
+
+Chosen option: Change tabindex value in main-website repo itself, because
```

Comment:
The option title should be more general I think and not refer to the main-website repo.